omniledger/service/struct.go

```diff
@@ -179,9 +178,10 @@ func (c *collectionDB) Store(t *StateChange) error {
 }
 
 func (c *collectionDB) GetValueContract(key []byte) ([]byte, []byte, error) {
-	return getValueContract(&roCollection{c.coll}, key)
+	return getValueContract(c, key)
 }
 
+// TODO this function can be merged with getValuesFromRecord
```

Comment:
Could you make an issue for this, and reference it here, and add label assistant, so we have useful jobs to give the assistants? Thx.

Reply:
done - https://github.com/dedis/cothority/issues/1363